app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDTOMockkUnitTest.kt

```diff
@@ -64,15 +50,42 @@ class DrinkDTOMockkUnitTest {
         mvm.drinkService = drinkService
     }
 
+    @Test
+    fun searchForCoke_returnsCoke(){
+        whenSearchForCoke()
+        thenResultContainsCoke()
+        thenVerifiedCokeFunctionsEnvoked()
+    }
+
+    @Test
+    fun searchForMariageFreres_returnsMariageFreres(){
+        whenSearchForMariageFreres()
+        thenResultContainsMariageFreres()
+        thenVerifiedMariageFreresFunctionsEnvoked()
+    }
+
+    @Test
+    fun searchForGarbage_ReturnsNothing(){
+        whenISearchForGarbage()
+        thenIGetZeroResults()
+    }
+
```

Comment:
Might be more of a personal preference then a "Best Practice", but organizing all of the tests, givens, whens, and thens makes for a more organized integration test.

Reply:
I agree... readable this way.